schemas/test/en/test_grand_calculated_summary_inside_repeating_section.json

```diff
@@ -216,6 +213,49 @@
                                     ]
                                 }
                             },
+                            "repeating_blocks": [
+                                {
+                                    "id": "cost-repeating-block-1",
+                                    "type": "ListRepeatingQuestion",
+                                    "question": {
+                                        "id": "cost-repeating-block-1-question",
+                                        "type": "General",
+                                        "title": {
+                                            "text": "What is the base monthly rate for {cost_name} for a single vehicle?",
+                                            "placeholders": [
+                                                {
+                                                    "placeholder": "cost_name",
+                                                    "value": {
+                                                        "source": "answers",
+                                                        "identifier": "cost-name"
+                                                    }
+                                                }
+                                            ]
+                                        },
+                                        "answers": [
+                                            {
+                                                "id": "cost-base",
```

Comment:
Should we keep the same convention here and call it `cost-repeating-block-1-answer` or `cost-repeating-block-1-answer-base`. Only a detail.

Reply:
Sure, not against changing these to be explicit, done same for the dynamic one